### PR TITLE
feat(pci.private.networks): add onboarding page for private network

### PR DESCRIPTION
**Pull request by vks-ovh**
LBaas L3 - Addition of onboarding page for private network
  ref: MANAGER-9470

Signed-off-by: vikash singh <vikash.singh@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9470
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

Addition of onboarding page for private networks as part of LBass L3 services.

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/private-networks/onboarding/onboarding.controller.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.user = coreConfig.getUser();
+  }
+
+  $onInit() {
+    this.loading = false;
+    this.illustration = illustration;
+    this.onboardingTrackPrefix = 'onboarding::';
+    this.guides = reduce(
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/onboarding/onboarding.controller.js:16]: **suggestion:** `Array.prototype.reduce` can be used.

File: packages/manager/modules/pci/src/projects/project/private-networks/private-networks.module.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ import deletePrivateNetwork from './delete';
 import vrack from './vrack';
 
 import empty from './empty';
+import onbarding from './onboarding';
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/private-networks.module.js:17]:
```suggestion
import onboarding from './onboarding';
```

File: packages/manager/modules/pci/src/projects/project/private-networks/onboarding/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "pci_projects_project_network_private": "Réseaux privés",
+  "pci_projects_project_network_private_vrack_empty": "Vous n'avez pas encore créé de réseau privé.",
+  "pci_projects_project_network_private_vrack_deploy": "Déployez des réseaux privés portés par le vRack d’OVH pour raccorder vos instances, où qu'elles se trouvent.",
+  "pci_projects_project_network_private_vrack_explanation_1": "Les réseaux privés sont créés à la demande et de manière instantanée. Ils garantissent une isolation de niveau 2 et peuvent accueillir un service DHCP pour fournir automatiquement l'adressage IP à vos instances. Chacun de ces réseaux peut être réduit à un emplacement ou s’étendre à l'ensemble des localisations de votre projet.",
+  "pci_projects_project_network_private_vrack_explanation_2": "Vos réseaux privés sont encapsulés dans votre vRack. Ils peuvent ainsi être connectés à d'autres solutions d’OVH attachées à ce même vRack, en utilisant un VLAN ID commun.",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/onboarding/translations/Messages_fr_FR.json:6]: d’OVH => d'OVHcloud

File: packages/manager/modules/pci/src/projects/project/private-networks/private-networks.module.js [hunk 4]
```diff
@@ -28,6 +29,7 @@ angular
     add,
     deletePrivateNetwork,
     vrack,
+    onbarding,
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/private-networks.module.js:32]:
```suggestion
    onboarding,
```

**Review by ghyenne — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/private-networks/onboarding/onboarding.controller.js (hunk 1 above)
Comment (vks-ovh) [reply packages/manager/modules/pci/src/projects/project/private-networks/onboarding/onboarding.controller.js:16]: yes, we can use. But we had used loadash at almost everywhere so thought to use the same.

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**